File: Femis Assignments/Morning Snacks/PerfectNumbers.java
public class PerfectNumbers {
    public static void main(String[] args) {

        for (int numbers = 1; numbers <= 1000; numbers++) {
            int sumOfFactors = 0;

            for (int factors = 1; factors <= numbers / 2; factors++) {
                if (numbers % factors == 0) {
                    sumOfFactors += factors;
                }
            }

            if (sumOfFactors == numbers) {
                System.out.println(numbers);
            }
        }
    }
}

File: Femis Assignments/Morning Snacks/perfect_numbers.js
for (let numbers = 1; numbers<= 1000; numbers++){
    let sum_of_factors =0;
    for (let factors = 1; factors<= numbers/2; factors++){
         if (numbers % factors == 0){
                sum_of_factors +=factors;
    }
    }
    if (sum_of_factors == numbers){
        console.log(numbers)
}
}
